src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java

```diff
@@ -100,7 +101,8 @@ public void updatePartitionsConfiguration(final Partition[] partitions) throws N
         }
     }
 
-    private Topology readTopology() throws NakadiRuntimeException,
+    @Override
+    public Topology getTopology() throws NakadiRuntimeException,
```

Comment:
why have you renamed it? As I see the actual functionality didn't change.

Reply:
it was different name - listPartitions(), now it is getTopology()